javascript/src/GherkinInMarkdownTokenMatcher.ts

```diff
@@ -130,7 +130,13 @@ export default class GherkinInMarkdownTokenMatcher implements ITokenMatcher<Toke
     let result = false
     if (token.line.startsWith('|')) {
       const tableCells = token.line.getTableCells()
-      if (this.isGfmTableSeparator(tableCells)) result = true
+      if (this.isGfmTableSeparator(tableCells)) {
+        // Maintain consistency with Python implementation
+        token.matchedType = TokenType.Comment
+        token.matchedText = undefined
+        token.matchedIndent=0
+        result = true
```

Comment:
I added these in an attempt to make the Python and Javascript versions consistent.  But the Javascript acceptance tests currently fail because of my comment above regarding `@cucumber/gherkin-streams`

If I try to remove the `matchedType` from the Python implementation, it fails - but I'm not sure how to resolve.

Guidance here would be appreciated.